### DISCUSSION TITLE
How to integrate Stripe?

### DISCUSSION THREAD

**Discussion by tornikeo** [zauberzeug/nicegui#862] [Q&A] [answered]
### Question

Is there any starter code or demo which lets me see how to use `stripe` payment API with `nicegui`?

Thanks!

**Comment by rodja** [comment, marked as answer]
Not yet. Do you want to integrate [Stripe Elements](https://stripe.com/en-de/payments/elements)? They should work quite nicely. It looks like they are just html and javascript which can easily be controlled with NiceGUI. A good way to get started may be to create a new [example in the NiceGUI repo](https://github.com/zauberzeug/nicegui/blob/main/CONTRIBUTING.md#examples), and push it as a draft pull request. That way the community can help you along the way and discuss individual parts and answer questions more easily.

**Reply by ZSchweyk** [reply]
I would definitely be interested in this! I just discovered NiceGUI and would find stripe elements really useful...

**Comment by frankvp11** [comment]
I've created PR #5039, I'd like to add the elements to that folder later!
